course_contents/2_intro_to_python/notes/02. Loops.md

```diff
@@ -32,50 +35,61 @@ while counter > 0: # the counter never gets changed, causing the conditional to
   print(f"{counter} seconds left until New Year!")
 ```
 
+**Extra resources**
+[While loops](https://www.teclado.com/30-days-of-python/python-30-day-8-while-loops#object-object-loops)
+
 ### The `for` loop
-- for loop runs a given number of times
-- general syntax for `for` loop is:
+- `for` loop runs a given number of times
+- general syntax for using the `for` loop is:
```

Comment:
What do you think of shortening a bit, deleting the 2nd bullet point, and maybe making reference to the iterating potential of the for loop?

### The `for` loop
- Runs a specific number of times
- Often used to go over all the elements of a collection

Reply:
I agree with suggested changes, changing it and resolving